### ISSUE TITLE
namingConvention doesn't detect nested

### ISSUE THREAD

**Issue by SimenB** [rossPatton/stylint#168]
```stylus
#kurvearkDialog {
    #utskriftDialog {
        width: 300px;
    }
}
```

The first one is reported, the second is not (don't ask why nested IDs... Could you add a rule on that? Overqualified)

**Comment by rossPatton** [comment]
... why nested ids? :smile:
